README.md

```diff
@@ -35,6 +35,49 @@ If you want to try out account tabs or device apps, edit `src/main.jsx`
 (it contains commented-out code for the different types of apps)
 and update the `index.html` file to match.
 
+## Making network requests
+
+Your Kraken app will be embedded within an iframe with a 
+[content security policy](https://cheatsheetseries.owasp.org/cheatsheets/Content_Security_Policy_Cheat_Sheet.html#detailed-csp-directives)
+set to only `connect-src` via the Kraken App Proxy. This means all outbound 
+network requests you make from your app must include some additional data.
```

Comment:
Not quite. That means only outbound requests to the Proxy are allowed. The fact we want extra data on top isn't enforced by CSP, it's our own requirements. You can probs just reword that, and the next section explains the rest.